### ISSUE TITLE
How are you supposed to use this?

### ISSUE THREAD

**Issue by martinezhermes** [ZekeSnider/Jared#78]
Following these steps

> Jared must be run a machine running macOS with an active messages account logged in. It has only been tested on 10.14 Mojave and later. It may work on old versions of macOS but this is not guaranteed as there may have been changes to the message database's schema. If you don't want Jared posting as you, it is recommended that you create a new Apple ID and user account on your mac, and run it in the background under that user. That way it's not using your main Apple ID.
> 
> Download Jared.app and move it to the applications folder.
> See [download section](https://github.com/ZekeSnider/Jared/issues/new#Download) at the top.
> 
> Run Jared.app, Allow Jared "Full Disk Access" in System Preferences.
> This is required because of macOS permissions that limit access to the messages database.
> 
> Grant access to automate Messages
> If you are running macOS Catalina or later, you will need to allow Jared access to automate the message app. This allows Jared to send messages.
> 
> (Optional) Allow contacts access
> You can optionally allow Jared access to your contacts so that it can provide and update names of contacts. The contacts are used to set/retrieve names only.
> 
> (Optional) Start REST API server
> If you wish to use the REST API, you will need to enable it. If you have a firewall enabled on your mac, you will see a dialog prompting you to allow Jared access to the port it is binding.

and nothing happens?, am I missing something?
tried /ping, /enable

how do you use this?

**Comment by ZekeSnider** [comment]
Hard for me to tell exactly without more info. If you start console.app and filter for Jared before starting it up, do you see any relevant logs there? Also which macOS version are you on? Have you tried to see if webhooks are working properly or not?

**Comment by MohammedFadin** [comment]
Hi Zake!

i think I'm having a similar problem to the OP. I can see Jared is processing things when filtering in Console but am I supposed to write myself for example with /help /ping? or I have to create a new contact called Jared?

**Comment by ZekeSnider** [comment]
Yeah you'll have to message yourself it you're running it under the same macOS user. If you want to have it as a separate contact, you should create a new user on your Mac and run it under there.

**Comment by MohammedFadin** [comment]
Thank you Zeke! It’s working wonderfully! Not quite sure what was the issue at the beginning when I started it. 

Question, if I have a new user for Jared, what is the best approach to forward certain messages to Jared? From my user to Jared’s user?

**Comment by ZekeSnider** [comment]
Yep exactly, you would just message the Apple ID address for the new user for any messages you wanted to go through the chat bot.

**Comment by MohammedFadin** [comment]
Hmm still thinking how to achieve this. My user for example doesn’t have Jared listening, and I don’t think there is a message forwarder app/tool to do this.
